src/redux/actions/auth/authActions.js

```diff
@@ -3,13 +3,24 @@ import { login, loginFailed, loginStarted, logout } from './authActionCreators';
 
 const BASE_URL = 'https://api.qa.openconceptlab.org/';
 
-const loginAction = ({ username, token }) => async (dispatch) => {
+const loginAction = ({ username, password }) => async (dispatch) => {
   try {
-    const url = `${BASE_URL}/users/${username}/`;
+    const data = {
+      username,
+      password,
+    };
+
+    // Get user's token
```

Comment:
An observation. Please be consistent with comments. But this good job.

Reply:
In regards to the above comment, can you take a look at this? https://improvingsoftware.com/2011/06/27/5-best-practices-for-commenting-your-code/

Reply:
@dkayiwa Thanks for the resource. I will work on it according.

Reply:
@alexmochu Thank you for your observation. I will read on the best practices and conventions for writing comments.